### PR TITLE
Check on quick strategy run using Mobile

### PR DESCRIPTION
**Pull request by thivya-deriv**
Desc: This is a test to run quick strategies using responsive 
Card: https://app.clickup.com/t/20696747/BOT-1738

### CODE REVIEW

**Review by markodonnell-deriv — APPROVED** [review]
Looks good

File: cypress/e2e/wip/dbot/sanity-tests/QuickStrategies.cy.js [hunk 1]
```diff
@@ -4,37 +4,61 @@ import BotBuilder from '../pageobjects/bot_builder_page'
...
-    //waiting for the bot to stop
-    cy.findByRole('button', { name: 'Run' }, { timeout: 120000 }).should(
-      'be.visible'
-    )
-    runPanel.transactionsTab.click()
-    //Verify Stake doubles after a loss
-    runPanel.runPanelScrollbar.scrollTo('bottom', { ensureScrollable: false })
-    runPanel.transactionAfterFirstLoss()
+  size.forEach((size) => {
+    it(`Run Martingale Quick Strategy ${size == 'small' ? 'mobile' : 'desktop'}`, () => {
```
Comment (abuzar-deriv) [inline cypress/e2e/wip/dbot/sanity-tests/QuickStrategies.cy.js:16]:
```suggestion
    it(`Run Martingale Quick Strategy on ${size == 'small' ? 'mobile' : 'desktop'}`, () => {
```

**Review by abuzar-deriv — CHANGES_REQUESTED** [review]
Minor text update needed.

**Review by abuzar-deriv — APPROVED** [review]
LGTM

**Pull request merged**